DESCRIPTION

```diff
@@ -22,7 +22,8 @@ Imports:
     stringr,
     jsonlite, 
     httr,
-    geoknife (>= 0.6.2)
+    geoknife (>= 0.6.2),
```

Comment:
this will need to be >0.10.3, but you'll get that when you `pull upstream master`